### PR TITLE
Update PayFineUI.java

### PR DESCRIPTION
**Pull request by dipanshu42**
Hey @navjeevan24  as my mediator please request my reviewer @dharam4  to review my code.

### CODE REVIEW

File: library/payfine/PayFineUI.java [hunk 1]
```diff
@@ -25,33 +29,35 @@ public void setState(UiState state)           //SeT_StAtE(uI_sTaTe state){
...
 		while (true) {
 			
-			switch (StAtE) {
+			switch (state) {            //StAtE
 			
 			case READY:
-				String Mem_Str = input("Swipe member card (press <enter> to cancel): ");
-				if (Mem_Str.length() == 0) {
-					CoNtRoL.CaNcEl();
+				String memstr = input("Swipe member card (press <enter> to cancel): ");     //Mem_Str
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:40]: you have done wrong here , It should be like memStr

File: library/payfine/PayFineUI.java [hunk 2]
```diff
@@ -25,33 +29,35 @@ public void setState(UiState state)           //SeT_StAtE(uI_sTaTe state){
...
+				if (memstr.length() == 0) {       //Mem_Str
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:41]: here also the memStr

File: library/payfine/PayFineUI.java [hunk 3]
```diff
@@ -25,33 +29,35 @@ public void setState(UiState state)           //SeT_StAtE(uI_sTaTe state){
...
 				}
 				break;
 				
 			case PAYING:
-				double AmouNT = 0;
-				String Amt_Str = input("Enter amount (<Enter> cancels) : ");
-				if (Amt_Str.length() == 0) {
-					CoNtRoL.CaNcEl();
+				double amount = 0;                                              //AmouNT
+				String amtstr = input("Enter amount (<Enter> cancels) : ");     //Amt_Str
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:56]: this should be amtStr

File: library/payfine/PayFineUI.java [hunk 4]
```diff
@@ -25,33 +29,35 @@ public void setState(UiState state)           //SeT_StAtE(uI_sTaTe state){
...
 				break;
 				
 			case PAYING:
-				double AmouNT = 0;
-				String Amt_Str = input("Enter amount (<Enter> cancels) : ");
-				if (Amt_Str.length() == 0) {
-					CoNtRoL.CaNcEl();
+				double amount = 0;                                              //AmouNT
+				String amtstr = input("Enter amount (<Enter> cancels) : ");     //Amt_Str 
+				if (amtstr.length() == 0) {                                    //Amt_Str
```
Comment (dharam4) [inline library/payfine/PayFineUI.java:57]: here  is also amtStr

**Comment by dipanshu42** [comment]
Ok, will look into it. Thnks for pointing this out.

**Review by dharam4 — APPROVED** [review]
Yes. Now it’s looking fine

**Comment by dipanshu42** [comment]
Thanks i'll merge later

**Pull request merged**